### ISSUE TITLE
error MSB3030: (NETCORE_ENGINEERING_TELEMETRY=Build) Could not copy the file

### ISSUE THREAD

**Issue by lewing** [dotnet/sdk#42964]
## Build Information
Build: https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_build/results?buildId=785882
Build error leg or test failing: Build / FullFramework: windows (x64) / Build
Pull request: https://github.com/dotnet/sdk/pull/42833

## Error Message

Fill the error message using [step by step known issues guidance](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/KnownIssueJsonStepByStep.md).

```json
{
  "ErrorMessage": "error MSB3030: (NETCORE_ENGINEERING_TELEMETRY=Build) Could not copy the file",
  "ErrorPattern": "",
  "BuildRetry": false,
  "ExcludeConsoleLog": false
}
```

 ### Known issue validation
**Build: :mag_right:** https://dev.azure.com/dnceng-public/public/_build/results?buildId=785882
**Error message validated:** `[error MSB3030: (NETCORE_ENGINEERING_TELEMETRY=Build) Could not copy the file`]
**Result validation:** :white_check_mark: Known issue matched with the provided build.
**Validation performed at:** 8/23/2024 3:00:11 AM UTC

### Report
|Build|Definition|Step Name|Console log|Pull Request|
|---|---|---|---|---|
|[2530143](https://dev.azure.com/dnceng/internal/_build/results?buildId=2530143)|dotnet-sdk|Build|[Log](https://dev.azure.com/dnceng/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/build/builds/2530143/logs/636)||
|[796353](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796353)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/796353/logs/85)|dotnet/sdk#43188|
|[796263](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796263)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/796263/logs/85)|dotnet/sdk#43183|
|[796019](https://dev.azure.com/dnceng-public/public/_build/results?buildId=796019)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/796019/logs/98)|dotnet/sdk#43070|
|[2529525](https://dev.azure.com/dnceng/internal/_build/results?buildId=2529525)|dotnet-sdk|Build|[Log](https://dev.azure.com/dnceng/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/build/builds/2529525/logs/387)||
|[795434](https://dev.azure.com/dnceng-public/public/_build/results?buildId=795434)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/795434/logs/96)|dotnet/sdk#42861|
|[795039](https://dev.azure.com/dnceng-public/public/_build/results?buildId=795039)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/795039/logs/95)|dotnet/sdk#43155|
|[794871](https://dev.azure.com/dnceng-public/public/_build/results?buildId=794871)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/794871/logs/88)|dotnet/sdk#43123|
|[2528005](https://dev.azure.com/dnceng/internal/_build/results?buildId=2528005)|dotnet-sdk|Build|[Log](https://dev.azure.com/dnceng/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/build/builds/2528005/logs/405)||
|[793820](https://dev.azure.com/dnceng-public/public/_build/results?buildId=793820)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/793820/logs/171)|dotnet/sdk#43128|
|[793372](https://dev.azure.com/dnceng-public/public/_build/results?buildId=793372)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/793372/logs/165)|dotnet/sdk#43113|
|[793359](https://dev.azure.com/dnceng-public/public/_build/results?buildId=793359)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/793359/logs/102)|dotnet/sdk#43111|
|[2527248](https://dev.azure.com/dnceng/internal/_build/results?buildId=2527248)|dotnet-sdk|Build|[Log](https://dev.azure.com/dnceng/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/build/builds/2527248/logs/357)||
|[793309](https://dev.azure.com/dnceng-public/public/_build/results?buildId=793309)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/793309/logs/107)|dotnet/sdk#43105|
|[792644](https://dev.azure.com/dnceng-public/public/_build/results?buildId=792644)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/792644/logs/112)|dotnet/sdk#43087|
|[792495](https://dev.azure.com/dnceng-public/public/_build/results?buildId=792495)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/792495/logs/131)|dotnet/sdk#42861|
|[792367](https://dev.azure.com/dnceng-public/public/_build/results?buildId=792367)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/792367/logs/100)|dotnet/sdk#43072|
|[2526192](https://dev.azure.com/dnceng/internal/_build/results?buildId=2526192)|dotnet-sdk|Build|[Log](https://dev.azure.com/dnceng/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/build/builds/2526192/logs/526)||
|[791173](https://dev.azure.com/dnceng-public/public/_build/results?buildId=791173)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/791173/logs/73)|dotnet/sdk#43024|
|[791078](https://dev.azure.com/dnceng-public/public/_build/results?buildId=791078)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/791078/logs/101)|dotnet/sdk#43020|
|[2525557](https://dev.azure.com/dnceng/internal/_build/results?buildId=2525557)|dotnet-sdk|Build|[Log](https://dev.azure.com/dnceng/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/build/builds/2525557/logs/399)||
|[790374](https://dev.azure.com/dnceng-public/public/_build/results?buildId=790374)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/790374/logs/102)|dotnet/sdk#43041|
|[790100](https://dev.azure.com/dnceng-public/public/_build/results?buildId=790100)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/790100/logs/87)|dotnet/sdk#43036|
|[789877](https://dev.azure.com/dnceng-public/public/_build/results?buildId=789877)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/789877/logs/114)|dotnet/sdk#43029|
|[789823](https://dev.azure.com/dnceng-public/public/_build/results?buildId=789823)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/789823/logs/207)|dotnet/sdk#43022|
|[789847](https://dev.azure.com/dnceng-public/public/_build/results?buildId=789847)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/789847/logs/99)|dotnet/sdk#43027|
|[789716](https://dev.azure.com/dnceng-public/public/_build/results?buildId=789716)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/789716/logs/90)|dotnet/sdk#42900|
|[2524943](https://dev.azure.com/dnceng/internal/_build/results?buildId=2524943)|dotnet-sdk|Build|[Log](https://dev.azure.com/dnceng/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/build/builds/2524943/logs/399)||
|[789440](https://dev.azure.com/dnceng-public/public/_build/results?buildId=789440)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/789440/logs/186)|dotnet/sdk#43015|
|[2524753](https://dev.azure.com/dnceng/internal/_build/results?buildId=2524753)|dotnet-sdk|Build (cblMariner20Fpm)|[Log](https://dev.azure.com/dnceng/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/build/builds/2524753/logs/453)||
|[2524677](https://dev.azure.com/dnceng/internal/_build/results?buildId=2524677)|dotnet-sdk|Build|[Log](https://dev.azure.com/dnceng/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/build/builds/2524677/logs/360)||
|[789116](https://dev.azure.com/dnceng-public/public/_build/results?buildId=789116)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/789116/logs/96)|dotnet/sdk#43007|
|[2524538](https://dev.azure.com/dnceng/internal/_build/results?buildId=2524538)|dotnet-sdk|Build|[Log](https://dev.azure.com/dnceng/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/build/builds/2524538/logs/577)||
|[2524492](https://dev.azure.com/dnceng/internal/_build/results?buildId=2524492)|dotnet-sdk|Build|[Log](https://dev.azure.com/dnceng/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/build/builds/2524492/logs/369)||
|[788918](https://dev.azure.com/dnceng-public/public/_build/results?buildId=788918)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/788918/logs/93)|dotnet/sdk#42699|
|[788072](https://dev.azure.com/dnceng-public/public/_build/results?buildId=788072)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/788072/logs/104)|dotnet/sdk#42972|
|[787931](https://dev.azure.com/dnceng-public/public/_build/results?buildId=787931)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/787931/logs/301)|dotnet/sdk#42998|
|[2523961](https://dev.azure.com/dnceng/internal/_build/results?buildId=2523961)|dotnet-sdk|Build|[Log](https://dev.azure.com/dnceng/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/build/builds/2523961/logs/365)||
|[787811](https://dev.azure.com/dnceng-public/public/_build/results?buildId=787811)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/787811/logs/106)|dotnet/sdk#42929|
|[787263](https://dev.azure.com/dnceng-public/public/_build/results?buildId=787263)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/787263/logs/173)|dotnet/sdk#42988|
|[787088](https://dev.azure.com/dnceng-public/public/_build/results?buildId=787088)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/787088/logs/151)|dotnet/sdk#42988|
|[786909](https://dev.azure.com/dnceng-public/public/_build/results?buildId=786909)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/786909/logs/151)|dotnet/sdk#42606|
|[786433](https://dev.azure.com/dnceng-public/public/_build/results?buildId=786433)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/786433/logs/107)|dotnet/sdk#42972|
|[786353](https://dev.azure.com/dnceng-public/public/_build/results?buildId=786353)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/786353/logs/87)|dotnet/sdk#42977|
|[2522485](https://dev.azure.com/dnceng/internal/_build/results?buildId=2522485)|dotnet-sdk|Build|[Log](https://dev.azure.com/dnceng/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/build/builds/2522485/logs/356)||
|[2522454](https://dev.azure.com/dnceng/internal/_build/results?buildId=2522454)|dotnet-sdk|Build (ubuntu2204DebPkg)|[Log](https://dev.azure.com/dnceng/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/build/builds/2522454/logs/719)||
|[2522356](https://dev.azure.com/dnceng/internal/_build/results?buildId=2522356)|dotnet-sdk|Build|[Log](https://dev.azure.com/dnceng/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/build/builds/2522356/logs/491)||
|[2522334](https://dev.azure.com/dnceng/internal/_build/results?buildId=2522334)|dotnet-sdk|Build (mariner20CrossArm)|[Log](https://dev.azure.com/dnceng/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/build/builds/2522334/logs/505)||
|[785882](https://dev.azure.com/dnceng-public/public/_build/results?buildId=785882)|dotnet/sdk|Build|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/785882/logs/171)|dotnet/sdk#42833|
#### Summary
|24-Hour Hit Count|7-Day Hit Count|1-Month Count|
|---|---|---|
|4|22|49|

**Comment by lewing** [comment]
@marcpopMSFT  is this a known problem in the sdk tests?

**Comment by marcpopMSFT** [comment]
@lewing yes it is. A workaround was put in two years ago for this but it was fairly hacky. @MiYanni undid that workaround on the path to simplifying the test infrastructure but that work is not done yet so this issue is still in play. We may want to think about a temporary fix to make sure this isn't impacting builds.

**Comment by marcpopMSFT** [comment]
I took another look at the binlog and I tracked down the potential issue. These files aren't being included in the default item include like I thought but because of an explicit include in the test directory.build.targets. I'm going to remove that include entirely and see what fails. Regardless, we should not copy the test assets into every single test publish folder as that's a lot of duplication for no reason.

https://github.com/dotnet/sdk/pull/43146

**Comment by marcpopMSFT** [comment]
Should be fixed in 9.0.1xx now with my linked PR.
